### ISSUE TITLE
Emulator Still crashes after game launch

### ISSUE THREAD

**Issue by shotgunfool** [yifanlu/PSXperia#6]
You can see the little popup that says updating for a split second and then the emulator still crashes

iv redownloaded the psxperia tool after you have made the change but still no luck,

Apk builds and installs fine but crashes on run

My System - Windows 7 32bit
My Phone - 2.3.3 Unrooted Locked Bootloader

Cheers.

**Comment by yifanlu** [comment]
Did you receive a warning about "unknown apk" when extracting that data? Either way, extract the data again, and post the console output.

Then, install again after reextracting and rebuilding, and when you're running it, open up logcat and copy the output from there.

Thank you.

**Comment by shotgunfool** [comment]
No Error warning about Unknown APK when extracting data,

Console Output /////////////////

[I] Verifying files
[I] Decoding values
11-Aug-2011 19:43:23 brut.androlib.res.AndrolibResources loadMainPkg
INFO: Loading resource table...
11-Aug-2011 19:43:23 brut.androlib.res.AndrolibResources loadMainPkg
INFO: Loaded.
11-Aug-2011 19:43:23 brut.androlib.res.AndrolibResources loadFrameworkPkg
INFO: Loading resource table from file: C:\Users\Oscar\apktool\framework\1.apk
11-Aug-2011 19:43:25 brut.androlib.res.AndrolibResources loadFrameworkPkg
INFO: Loaded.
11-Aug-2011 19:43:25 brut.androlib.res.AndrolibResources decode
INFO: Decoding file-resources...
11-Aug-2011 19:43:25 brut.androlib.res.AndrolibResources decode
INFO: Decoding values_/_ XMLs...
11-Aug-2011 19:43:25 brut.androlib.res.AndrolibResources decode
INFO: Done.
[I] Extracting APK
[I] Extracting ZIP file: C:\Users\Oscar\Desktop\com.sony.playstation.ncea00344_1
.apk to: C:\Users\Oscar\Desktop\Extracted Data
[D] Done extracting ZIP.
[I] Extracting ZPAKS
[I] Extracting ZIP file: C:\Users\Oscar\Desktop\Extracted Data\assets\NCEA00344_
1_1.zpak to: C:\Users\Oscar\Desktop\Extracted Data\assets\ZPAK
[D] Done extracting ZIP.
[I] Extracting ZIP file: C:\Users\Oscar\Desktop\NCEA00344_1_1.zpak to: C:\Users\
Oscar\Desktop\Extracted Data\ZPAK
[D] Done extracting ZIP.
[I] Removing unneeded files.
[I] Adding new files.
[I] Patching XML strings with tags.
[D] Replacing strings in XML data.
[D] Writing to temporary file: C:\Users\Oscar\Desktop\Extracted Data\assets\Andr
oidManifest.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: C:\Users\Oscar\Desktop\Extracted Data\assets\ZPAK
\metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: C:\Users\Oscar\Desktop\Extracted Data\res\values\
strings.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: C:\Users\Oscar\Desktop\Extracted Data\ZPAK\metada
ta.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Done with string replacement.
[I] Done. 

////////////////////

Currently Rebuilding and going to install,

how do i get a logcat report, im not rooted i dont normally do any beta testing so not sure where error reports are.

Cheers

**Comment by shotgunfool** [comment]
Here is the console output on the construction of the apk and the zpak file

///////////////////////

[I] PSXperiaTool initialized, outputting to: C:\Users\Oscar\Desktop\Fifa 98 Outp
ut File
[I] Starting build.
[I] Checking to make sure all files are there.
[D] Done checking data.
[I] Creating temporary directory.
[D] Created temporary directory at, ..psxperia.726
[I] Copying icon if needed.
[I] Replacing strings.
[D] Replacing strings in XML data.
[D] Writing to temporary file: ..psxperia.726\assets\AndroidManifest.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ..psxperia.726\assets\ZPAK\metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ..psxperia.726\res\values\strings.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ..psxperia.726\ZPAK\metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Done with string replacement.
[D] Done replacing strings.
[I] Generating PSImage.
[I] Beginning PSImage compression.
[I] Generating image TOC.
[D] Done generating PSImage
[I] Generating ZPAK.
[I] Generating zpak file from directory ..psxperia.726\ZPAK with compression =
false
[D] Done with ZPAK creation.
[D] Done generating ZPAK at ..psxperia.726\NCXA12312_1.zpak
[I] Generating default ZPAK.
[I] Generating zpak file from directory ..psxperia.726\assets\ZPAK with compres
sion = true
[D] Done with ZPAK creation.
[D] Done generating default ZPAK at ..psxperia.726\assets\NCXA12312_1.zpak
[I] Done processing, generating output.
[D] Moving files around.
[D] Running command: [aapt package -f -F C:\Users\Oscar\Desktop\Fifa 98 Output F
ile\com.sony.playstation.NCXA12312_1.apk -S ..psxperia.726\res -M ..psxperia.7
26\assets\AndroidManifest.xml -I .\android-framework.jar ..psxperia.726]
[D] Running command: [jarsigner -keystore .\signApk -storepass password -keypass
 password -signedjar C:\Users\Oscar\Desktop\Fifa 98 Output File\com.sony.playsta
tion.NCXA12312_1.apk.signed C:\Users\Oscar\Desktop\Fifa 98 Output File\com.sony.
playstation.NCXA12312_1.apk signPSXperia]
[I] Done.
[I] APK file: C:\Users\Oscar\Desktop\Fifa 98 Output File\com.sony.playstation.NC
XA12312_1.apk
[I] Data dir: C:\Users\Oscar\Desktop\Fifa 98 Output File\data\com.sony.playstati
on.NCXA12312_1\files\content
[I] Deleting temporary directory
[I] Done.

////////////////////////////////

And currently transferring the data to my memory card ready to install

**Comment by shotgunfool** [comment]
Logcat report of the crash - 

///////////////////////////////

D/alogcat ( 1364): stopping ...
D/alogcat ( 1364): stopped
D/alogcat ( 1364): starting ...
V/alogcat ( 1364): save instance
V/alogcat ( 1364): paused
D/alogcat ( 1364): stopping ...
D/alogcat ( 1364): stopped
I/ZJavaActivity( 1394): onCreate
D/ZJavaActivity( 1394): library=/data/data/com.sony.playstation.NCXA12312_1/lib/libjava-activity-wrapper.so
D/dalvikvm( 1394): Trying to load lib /data/data/com.sony.playstation.NCXA12312_1/lib/libjava-activity-wrapper.so 0x2afc7e20
D/dalvikvm( 1394): Added shared lib /data/data/com.sony.playstation.NCXA12312_1/lib/libjava-activity-wrapper.so 0x2afc7e20
I/ZJavaActivity( 1394): CreateContentView()
I/ZJavaActivity( 1394): MotionEvent has getSource()!
D/ZJavaActivity( 1394): sourceDir=/data/app/com.sony.playstation.NCXA12312_1-1.apk
D/ZJavaActivity( 1394): publicSourceDir=/data/app/com.sony.playstation.NCXA12312_1-1.apk
D/ZJavaActivity( 1394): processName=com.sony.playstation.NCXA12312_1
D/ZJavaActivity( 1394): dataDir=/data/data/com.sony.playstation.NCXA12312_1
D/ZJavaActivity( 1394): PackageName=com.sony.playstation.NCXA12312_1
D/dalvikvm( 1394): Trying to load lib /data/data/com.sony.playstation.NCXA12312_1/lib/libjava-activity.so 0x2afc7e20
D/dalvikvm( 1394): Added shared lib /data/data/com.sony.playstation.NCXA12312_1/lib/libjava-activity.so 0x2afc7e20
I/ActivityManager(  195): Displayed com.sony.playstation.NCXA12312_1/com.sony.android.psone.BootActivity: +381ms
I/ZJavaActivity( 1394): onGlobalLayout()
I/ZJavaActivity( 1394): surfaceCreated()
I/ZJavaActivity( 1394): surfaceChanged()
I/ZJavaActivity( 1394): onGlobalLayout()
W/IInputConnectionWrapper(  365): showStatusIcon on inactive InputConnection
I/PackageIntentReceiver( 1137): Package com.sony.playstation.NCXA12312_1 Launched
I/PackageIntentReceiver( 1146): Package com.sony.playstation.NCXA12312_1 Launched
V/Z-System( 1394): Application loaded: 
W/dalvikvm( 1394): dvmFindClassByName rejecting 'com/sonyericsson/zsystem/jni/ZJavaActivityInput'
W/dalvikvm( 1394): dvmFindClassByName rejecting 'com/sonyericsson/zsystem/jni/ZJavaActivitySensor'
D/ZJavaActivity( 1394): ICON ID=2130837504 TITLE ID=2130968613 DESC ID=2130968602
D/lights  (  195): set_light_attention: color=0x00ffffff, mode=2
D/lights  (  195): set_light_attention: color=0x00000000, mode=0
D/dalvikvm(  195): GC_EXPLICIT freed 498K, 40% free 6577K/10951K, external 4014K/4137K, paused 71ms
V/Z-System( 1394): --------------------------------------------------------------------
V/Z-System( 1394): zcHandleOutputOpenHandles - Handle created in file 'zcCriticalSectionCreate', line 298, was not closed.
V/Z-System( 1394): zcHandleOutputOpenHandles - Handle created in file 'zplatform/sourcecode/zplatform/zpenumerate.cpp', line 1299, was not closed.
V/Z-System( 1394): --------------------------------------------------------------------
E/JavaBinder(  304): !!! FAILED BINDER TRANSACTION !!!
E/kernel  (  115): [  154.841339] binder: 1394: binder_alloc_buf, no vma
D/kernel  (  115): [  154.841369] binder: 304:436 transaction failed 29201, size 132-4
E/InputDispatcher(  195): channel '2b2f9310 com.sony.playstation.NCXA12312_1/com.sony.android.psone.BootActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0x8
E/InputDispatcher(  195): channel '2b2f9310 com.sony.playstation.NCXA12312_1/com.sony.android.psone.BootActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
D/kernel  (  115): [  154.933563] binder: 304:304 transaction failed 29189, size 92-0
D/dalvikvm(  365): GC_CONCURRENT freed 2099K, 58% free 3462K/8199K, external 9875K/11756K, paused 6ms+3ms
D/dalvikvm(  365): GC_CONCURRENT freed 194K, 58% free 3488K/8199K, external 10753K/12315K, paused 2ms+3ms
W/InputConnectionWrapper.ICC(  304): Timed out waiting on IInputContextCallback
D/dalvikvm(  195): GC_EXPLICIT freed 120K, 41% free 6519K/10951K, external 3365K/4137K, paused 96ms
V/alogcat ( 1364): created
D/alogcat ( 1364): canceling periodic saves
V/alogcat ( 1364): started
V/alogcat ( 1364): resumed
D/alogcat ( 1364): starting ...
W/ResourceType(  195): Skipping entry 0x7f04001a in package table 0 because it is not complex!
W/ResourceType(  195): Skipping entry 0x7f04003c in package table 0 because it is not complex!
I/ActivityManager(  195): Displayed org.jtb.alogcat/.LogActivity: +455ms
D/dalvikvm( 1364): GC_CONCURRENT freed 467K, 52% free 2896K/5959K, external 4141K/4198K, paused 6ms+3ms
I/SecureClock(  576): Starting SecureClockService
I/SecureClock(  576): The listener for network connectivity is set up
I/SecureClock(  576): Generate an initial secure clock value.
I/SecureClock(  576): Network not connected
I/SecureClock(  576): network state changed
I/SecureClock(  576): The phone is now connected with the network again!
I/SecureClock(  576): fetchNTPTime started
I/SecureClock(  576): Enter NTP request guard
I/SecureClock(  576): Ntp time fetching triggered.
I/SecureClock(  576): Trying to fetch NTP time from the server: xtra1.gpsonextra.net
I/SecureClock(  576): Retrieved Ntp time:1313089073
I/SecureClock(  576): Stopping parent service
I/SecureClock(  576): Exit NTP request guard
W/KeyCharacterMap( 1364): No keyboard for id 0
W/KeyCharacterMap( 1364): Using default keymap: /system/usr/keychars/qwerty.kcm.bin

/////////////////////////////////

Cheers.

**Comment by Levistras** [comment]
I'm getting the same issue as well.  
I'm on an R800a running 2.3.3, build 3.0.1.A.0.145.

I don't see any issues in the Java GUI when creating the output folder and the APK.  When I run the App on device (installs fine) you see the "Updating..." box very quickly, and then the screen goes black.  A second or two later I'm returned to the home screen.

Here's my logcat from the device side:

I/ActivityManager(  195): Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.sony.playstation.NCXA94163_1/com.sony.android.psone.BootActivity } from pid 22383
I/ActivityManager(  195): Start proc com.sony.playstation.NCXA94163_1 for activity com.sony.playstation.NCXA94163_1/com.sony.android.psone.BootActivity: pid=22440 uid=10170 gids={3003, 1015}
I/ZJavaActivity(22440): onCreate
D/ZJavaActivity(22440): library=/data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity-wrapper.so
D/dalvikvm(22440): Trying to load lib /data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity-wrapper.so 0x2afc7be8
D/dalvikvm(22440): Added shared lib /data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity-wrapper.so 0x2afc7be8
I/ZJavaActivity(22440): CreateContentView()
I/ZJavaActivity(22440): MotionEvent has getSource()!
D/ZJavaActivity(22440): sourceDir=/data/app/com.sony.playstation.NCXA94163_1-1.apk
D/ZJavaActivity(22440): publicSourceDir=/data/app/com.sony.playstation.NCXA94163_1-1.apk
D/ZJavaActivity(22440): processName=com.sony.playstation.NCXA94163_1
D/ZJavaActivity(22440): dataDir=/data/data/com.sony.playstation.NCXA94163_1
D/ZJavaActivity(22440): PackageName=com.sony.playstation.NCXA94163_1
D/dalvikvm(22440): Trying to load lib /data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity.so 0x2afc7be8
D/dalvikvm(22440): Added shared lib /data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity.so 0x2afc7be8
I/ZJavaActivity(22440): onGlobalLayout()
I/ZJavaActivity(22440): surfaceCreated()
I/ZJavaActivity(22440): surfaceChanged()
I/ActivityManager(  195): Displayed com.sony.playstation.NCXA94163_1/com.sony.android.psone.BootActivity: +375ms
I/ZJavaActivity(22440): onGlobalLayout()
I/PackageIntentReceiver(22165): Package com.sony.playstation.NCXA94163_1 Launched
I/PackageIntentReceiver(22182): Package com.sony.playstation.NCXA94163_1 Launched
I/ZJavaActivity(22440): onGlobalLayout()
W/dalvikvm(22440): dvmFindClassByName rejecting 'com/sonyericsson/zsystem/jni/ZJavaActivityInput'
W/dalvikvm(22440): dvmFindClassByName rejecting 'com/sonyericsson/zsystem/jni/ZJavaActivitySensor'
D/ZJavaActivity(22440): ICON ID=2130837504 TITLE ID=2130968613 DESC ID=2130968602
D/lights  (  195): set_light_attention: color=0x00ffffff, mode=2
D/lights  (  195): set_light_attention: color=0x00000000, mode=0
I/ZJavaActivity(22440): surfaceDestroyed()
I/ActivityManager(  195): Process com.sony.playstation.NCXA94163_1 (pid 22440) has died.
I/WindowManager(  195): WIN DEATH: Window{2b3625e8 SurfaceView paused=false}
I/WindowManager(  195): WIN DEATH: Window{2b456d60 com.sony.playstation.NCXA94163_1/com.sony.android.psone.BootActivity paused=false}
W/StoreFront(  449): No MCC and MNC for the sim. No SIM card?
E/kernel  (  115): [38538.285430] init: untracked pid 22439 exited
D/dalvikvm(  449): GC_EXPLICIT freed 822K, 47% free 5211K/9735K, external 3009K/3425K, paused 100ms
W/StoreFront(  449): No MCC and MNC for the sim. No SIM card?
E/StoreFront(  449): Not ok from server: 502
D/dalvikvm(  449): GC_CONCURRENT freed 383K, 44% free 5478K/9735K, external 2932K/3662K, paused 2ms+5ms
W/ActivityManager(  195): Activity destroy timeout for HistoryRecord{2b574a70 com.sony.playstation.NCXA94163_1/com.sony.android.psone.BootActivity}
D/dalvikvm(22278): GC_EXPLICIT freed 4K, 53% free 2571K/5379K, external 1685K/2133K, paused 119ms
D/lights  (  195): set_light_backlight: brightness=243
D/lights  (  195): set_light_backlight: brightness=231
D/lights  (  195): set_light_backlight: brightness=219
D/lights  (  195): set_light_backlight: brightness=208
D/lights  (  195): set_light_backlight: brightness=196
D/lights  (  195): set_light_backlight: brightness=184
D/lights  (  195): set_light_backlight: brightness=172
D/lights  (  195): set_light_backlight: brightness=161
D/lights  (  195): set_light_backlight: brightness=149
D/lights  (  195): set_light_backlight: brightness=137
D/lights  (  195): set_light_backlight: brightness=125
D/lights  (  195): set_light_backlight: brightness=114
D/lights  (  195): set_light_backlight: brightness=102
D/lights  (  195): set_light_backlight: brightness=90
D/lights  (  195): set_light_backlight: brightness=78
D/lights  (  195): set_light_backlight: brightness=67
D/lights  (  195): set_light_backlight: brightness=55
D/lights  (  195): set_light_backlight: brightness=43
D/lights  (  195): set_light_backlight: brightness=31
D/lights  (  195): set_light_backlight: brightness=20
D/lights  (  195): set_light_backlight: brightness=35
D/lights  (  195): set_light_backlight: brightness=51
D/lights  (  195): set_light_backlight: brightness=67
D/lights  (  195): set_light_backlight: brightness=82
D/lights  (  195): set_light_backlight: brightness=98
D/lights  (  195): set_light_backlight: brightness=113
D/lights  (  195): set_light_backlight: brightness=129
D/lights  (  195): set_light_backlight: brightness=145
D/lights  (  195): set_light_backlight: brightness=161
D/lights  (  195): set_light_backlight: brightness=176
D/lights  (  195): set_light_backlight: brightness=192
D/lights  (  195): set_light_backlight: brightness=208
D/lights  (  195): set_light_backlight: brightness=223
D/lights  (  195): set_light_backlight: brightness=239
D/lights  (  195): set_light_backlight: brightness=255
D/dalvikvm(22383): GC_CONCURRENT freed 24K, 49% free 3290K/6407K, external 10748K/11177K, paused 2ms+2ms
I/ActivityManager(  195): Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.sony.playstation.NCXA94163_1/com.sony.android.psone.BootActivity } from pid 22383
I/ActivityManager(  195): Start proc com.sony.playstation.NCXA94163_1 for activity com.sony.playstation.NCXA94163_1/com.sony.android.psone.BootActivity: pid=22453 uid=10170 gids={3003, 1015}
I/ZJavaActivity(22453): onCreate
D/ZJavaActivity(22453): library=/data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity-wrapper.so
D/dalvikvm(22453): Trying to load lib /data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity-wrapper.so 0x2afc8048
D/dalvikvm(22453): Added shared lib /data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity-wrapper.so 0x2afc8048
I/ZJavaActivity(22453): CreateContentView()
I/ZJavaActivity(22453): MotionEvent has getSource()!
D/ZJavaActivity(22453): sourceDir=/data/app/com.sony.playstation.NCXA94163_1-1.apk
D/ZJavaActivity(22453): publicSourceDir=/data/app/com.sony.playstation.NCXA94163_1-1.apk
D/ZJavaActivity(22453): processName=com.sony.playstation.NCXA94163_1
D/ZJavaActivity(22453): dataDir=/data/data/com.sony.playstation.NCXA94163_1
D/ZJavaActivity(22453): PackageName=com.sony.playstation.NCXA94163_1
D/dalvikvm(22453): Trying to load lib /data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity.so 0x2afc8048
D/dalvikvm(22453): Added shared lib /data/data/com.sony.playstation.NCXA94163_1/lib/libjava-activity.so 0x2afc8048
I/ZJavaActivity(22453): onGlobalLayout()
I/ZJavaActivity(22453): surfaceCreated()
I/ZJavaActivity(22453): surfaceChanged()
I/ActivityManager(  195): Displayed com.sony.playstation.NCXA94163_1/com.sony.android.psone.BootActivity: +380ms
I/ZJavaActivity(22453): onGlobalLayout()
I/PackageIntentReceiver(22165): Package com.sony.playstation.NCXA94163_1 Launched
I/PackageIntentReceiver(22182): Package com.sony.playstation.NCXA94163_1 Launched
I/ZJavaActivity(22453): onGlobalLayout()
W/dalvikvm(22453): dvmFindClassByName rejecting 'com/sonyericsson/zsystem/jni/ZJavaActivityInput'
W/dalvikvm(22453): dvmFindClassByName rejecting 'com/sonyericsson/zsystem/jni/ZJavaActivitySensor'
D/ZJavaActivity(22453): ICON ID=2130837504 TITLE ID=2130968613 DESC ID=2130968602
D/lights  (  195): set_light_attention: color=0x00ffffff, mode=2
D/lights  (  195): set_light_attention: color=0x00000000, mode=0
I/ZJavaActivity(22453): surfaceDestroyed()
I/ActivityManager(  195): Process com.sony.playstation.NCXA94163_1 (pid 22453) has died.
I/WindowManager(  195): WIN DEATH: Window{2b76a358 SurfaceView paused=false}
I/WindowManager(  195): WIN DEATH: Window{2b5c7c50 com.sony.playstation.NCXA94163_1/com.sony.android.psone.BootActivity paused=false}
D/lights  (  195): set_light_buttons: brightness=255
D/kernel  (  115): [38575.933898] max17040 0-0036: batt: 96%, 4182 mV

**Comment by yifanlu** [comment]
Can you update to the latest JAR (https://github.com/downloads/yifanlu/PSXperia/psxperiatool-beta-2.zip) and try again. If it still exists, send me your APK.

**Comment by shotgunfool** [comment]
Downloading Now 

Give Me 5 Mins

Oh and just FYI thanks for your amazing work

**Comment by shotgunfool** [comment]
Got Cannot Build APK Error

**Comment by yifanlu** [comment]
Can you post the whole output?

**Comment by shotgunfool** [comment]
there isnt an output, the cmd prompt window has been removed

just trying to rebuild without changing any of the details to see if that has any effect

*Update

APK Build when not changing any of the details in the convert section,

just copying files to phone and going to install in 30 seconds

*Update

APK installs fine

Still Crashes On Load

Just going to get a logcat and upload my apk

**Comment by shotgunfool** [comment]
Logcat

/////////////////////////////

D/alogcat ( 2843): stopping ...
D/alogcat ( 2843): stopped
D/alogcat ( 2843): starting ...
V/alogcat ( 2843): save instance
V/alogcat ( 2843): paused
D/alogcat ( 2843): stopping ...
D/alogcat ( 2843): stopped
D/dalvikvm(  365): GC_CONCURRENT freed 1631K, 58% free 3654K/8583K, external 12059K/14060K, paused 2ms+4ms
I/ZJavaActivity( 2887): onCreate
D/ZJavaActivity( 2887): library=/data/data/com.sony.playstation.NCXA00000_1/lib/libjava-activity-wrapper.so
D/dalvikvm( 2887): Trying to load lib /data/data/com.sony.playstation.NCXA00000_1/lib/libjava-activity-wrapper.so 0x2afc6dc0
D/dalvikvm( 2887): Added shared lib /data/data/com.sony.playstation.NCXA00000_1/lib/libjava-activity-wrapper.so 0x2afc6dc0
I/ZJavaActivity( 2887): CreateContentView()
I/ZJavaActivity( 2887): MotionEvent has getSource()!
D/ZJavaActivity( 2887): sourceDir=/data/app/com.sony.playstation.NCXA00000_1-1.apk
D/ZJavaActivity( 2887): publicSourceDir=/data/app/com.sony.playstation.NCXA00000_1-1.apk
D/ZJavaActivity( 2887): processName=com.sony.playstation.NCXA00000_1
D/ZJavaActivity( 2887): dataDir=/data/data/com.sony.playstation.NCXA00000_1
D/ZJavaActivity( 2887): PackageName=com.sony.playstation.NCXA00000_1
D/dalvikvm( 2887): Trying to load lib /data/data/com.sony.playstation.NCXA00000_1/lib/libjava-activity.so 0x2afc6dc0
D/dalvikvm( 2887): Added shared lib /data/data/com.sony.playstation.NCXA00000_1/lib/libjava-activity.so 0x2afc6dc0
I/ActivityManager(  195): Displayed com.sony.playstation.NCXA00000_1/com.sony.android.psone.BootActivity: +352ms
I/ZJavaActivity( 2887): onGlobalLayout()
I/ZJavaActivity( 2887): surfaceCreated()
I/ZJavaActivity( 2887): surfaceChanged()
I/ZJavaActivity( 2887): onGlobalLayout()
W/IInputConnectionWrapper(  365): showStatusIcon on inactive InputConnection
I/PackageIntentReceiver( 2740): Package com.sony.playstation.NCXA00000_1 Launched
I/PackageIntentReceiver( 2757): Package com.sony.playstation.NCXA00000_1 Launched
V/Z-System( 2887): Application loaded: 
W/dalvikvm( 2887): dvmFindClassByName rejecting 'com/sonyericsson/zsystem/jni/ZJavaActivityInput'
W/dalvikvm( 2887): dvmFindClassByName rejecting 'com/sonyericsson/zsystem/jni/ZJavaActivitySensor'
D/ZJavaActivity( 2887): ICON ID=2130837504 TITLE ID=2130968613 DESC ID=2130968602
D/lights  (  195): set_light_attention: color=0x00ffffff, mode=2
D/lights  (  195): set_light_attention: color=0x00000000, mode=0
D/dalvikvm(  195): GC_EXPLICIT freed 513K, 43% free 6650K/11591K, external 3779K/4137K, paused 68ms
V/Z-System( 2887): --------------------------------------------------------------------
V/Z-System( 2887): zcHandleOutputOpenHandles - Handle created in file 'zcCriticalSectionCreate', line 298, was not closed.
V/Z-System( 2887): zcHandleOutputOpenHandles - Handle created in file 'zplatform/sourcecode/zplatform/zpenumerate.cpp', line 1299, was not closed.
V/Z-System( 2887): --------------------------------------------------------------------
E/InputDispatcher(  195): channel '2b2c2ec8 com.sony.playstation.NCXA00000_1/com.sony.android.psone.BootActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0x8
E/InputDispatcher(  195): channel '2b2c2ec8 com.sony.playstation.NCXA00000_1/com.sony.android.psone.BootActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
E/kernel  (  115): [ 1979.363708] binder: release proc 2887, transaction 87284, not freed
D/kernel  (  115): [ 1979.420013] binder: 1413:1413 transaction failed 29189, size 92-0
W/InputConnectionWrapper.ICC( 1413): Timed out waiting on IInputContextCallback
D/dalvikvm(  195): GC_EXPLICIT freed 89K, 44% free 6596K/11591K, external 3365K/4137K, paused 92ms
V/alogcat ( 2843): created
D/alogcat ( 2843): canceling periodic saves
V/alogcat ( 2843): started
V/alogcat ( 2843): resumed
D/alogcat ( 2843): starting ...
I/ActivityManager(  195): Displayed org.jtb.alogcat/.LogActivity: +441ms
D/dalvikvm( 2843): GC_CONCURRENT freed 844K, 54% free 2968K/6407K, external 2797K/3355K, paused 2ms+3ms

///////////////////////////////

Uploading APK now

**Comment by shotgunfool** [comment]
My APK

-removed-

**Comment by yifanlu** [comment]
That was generated by an old build. Please download the newest version: https://github.com/downloads/yifanlu/PSXperia/psxperia-beta-2.zip

I can tell from the APK that it contains the same bug fixed in #2

**Comment by shotgunfool** [comment]
Yeah i just noticed on xda you posted the new link ;) 

generating the PSImage now

will post back when done and installed

**Comment by Levistras** [comment]
Ditto, generating now.  Will report back in a few minutes.

**Comment by evo6** [comment]
Hi,

I was also getting the same issue, what every you put in that data folder now works for me :). The crashing has stopped when the emulator was starting.

Thanks so much yifanlu!

**Comment by shotgunfool** [comment]
oh and just so i know, do you have to change the Title ID: field so the 5 0's are random numbers?

**Comment by yifanlu** [comment]
You don't HAVE to, but every game needs a unique id, so you might as well.

**Comment by shotgunfool** [comment]
ok, well i just tried without changing and left it as it was and got 

cannot build APK

so changed the numbers and rebuilding

**Comment by yifanlu** [comment]
Open command prompt, "cd" to the folder with the JAR. Then type in "java -jar PSXperiaTool.jar". Now you will get a console output for everything. Do whatever you did before, and copy that output for me.

**Comment by shotgunfool** [comment]
Rebuilding now, cmd prompt is up

**Comment by shotgunfool** [comment]
I can tell what the error is from this its saying that the image file im using isnt a .PNG file, it is but could it be that the image is too big in dimensions for it to read it?

here is the output anyway

////////////////////////////

[I] PSXperiaTool initialized, outputting to: C:\Users\Oscar\Desktop\Fifa 98 Outp
ut File
[I] Starting build.
[I] Checking to make sure all files are there.
[D] Done checking data.
[I] Creating temporary directory.
[D] Created temporary directory at, ..psxperia.444
[I] Copying icon if needed.
[D] Done copying icon from C:\Users\Oscar\Pictures\fifa 98 free download.png
[I] Replacing strings.
[D] Replacing strings in XML data.
[D] Writing to temporary file: ..psxperia.444\assets\AndroidManifest.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ..psxperia.444\assets\ZPAK\metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ..psxperia.444\res\values\strings.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ..psxperia.444\ZPAK\metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Done with string replacement.
[D] Done replacing strings.
[I] Generating PSImage.
[I] Beginning PSImage compression.
[I] Generating image TOC.
[D] Done generating PSImage
[I] Generating ZPAK.
[I] Generating zpak file from directory ..psxperia.444\ZPAK with compression =
false
[D] Done with ZPAK creation.
[D] Done generating ZPAK at ..psxperia.444\NCXA12121_1.zpak
[I] Generating default ZPAK.
[I] Generating zpak file from directory ..psxperia.444\assets\ZPAK with compres
sion = true
[D] Done with ZPAK creation.
[D] Done generating default ZPAK at ..psxperia.444\assets\NCXA12121_1.zpak
[I] Done processing, generating output.
[D] Moving files around.
[D] Running command: [.\aapt-windows.exe package -f -F C:\Users\Oscar\Desktop\Fi
fa 98 Output File\com.sony.playstation.NCXA12121_1.apk.unsigned -S ..psxperia.4
44\res -M ..psxperia.444\assets\AndroidManifest.xml -I .\android-framework.jar
..psxperia.444]
[D] libpng error: Not a PNG file
[D] ERROR: Failure processing PNG image ..psxperia.444\res\drawable\icon.png
[D] ..psxperia.444\res\values\public.xml:3: error: Public symbol drawable/contr
oller_icon declared here is not defined.
[D] ..psxperia.444\res\values\public.xml:4: error: Public symbol drawable/custo
m_horizontal declared here is not defined.
[D] ..psxperia.444\res\values\public.xml:5: error: Public symbol drawable/icon
declared here is not defined.
[D] ..psxperia.444\res\values\public.xml:6: error: Public symbol drawable/title
_icon declared here is not defined.
[E] Cannot build APK!

//////////////////////////////

**Comment by yifanlu** [comment]
Maybe. I use a 170x170 image. Try resizing it.

**Comment by shotgunfool** [comment]
Just Resized, waiting for a rebuild will post results

**Comment by Levistras** [comment]
Mine compiled just fine (as it was before).  Loading over to the device to attempt firing it up.

Here's the log from the build:

[I] Verifying files
[I] Decoding values
Aug 11, 2011 5:02:01 PM brut.androlib.res.AndrolibResources loadMainPkg
INFO: Loading resource table...
Aug 11, 2011 5:02:02 PM brut.androlib.res.AndrolibResources loadMainPkg
INFO: Loaded.
Aug 11, 2011 5:02:02 PM brut.androlib.res.AndrolibResources loadFrameworkPkg
INFO: Loading resource table from file: C:\Users\fb\apktool\framework\1.apk
Aug 11, 2011 5:02:18 PM brut.androlib.res.AndrolibResources loadFrameworkPkg
INFO: Loaded.
Aug 11, 2011 5:02:18 PM brut.androlib.res.AndrolibResources decode
INFO: Decoding file-resources...
Aug 11, 2011 5:02:18 PM brut.androlib.res.AndrolibResources decode
INFO: Decoding values_/_ XMLs...
Aug 11, 2011 5:02:18 PM brut.androlib.res.AndrolibResources decode
INFO: Done.
[I] Extracting APK
[I] Extracting ZIP file: D:\XperiaTool\com.sony.playstation.ncua94900_1.apk to:
D:\XperiaTool.\data
[D] Done extracting ZIP.
[I] Extracting ZPAKS
[I] Extracting ZIP file: D:\XperiaTool.\data\assets\NCUA94900_1_1.zpak to: D:\X
periaTool.\data\assets\ZPAK
[D] Done extracting ZIP.
[I] Extracting ZIP file: D:\XperiaTool\com.sony.playstation.NCUA94900_1\files\co
ntent\NCUA94900_1_1.zpak to: D:\XperiaTool.\data\ZPAK
[D] Done extracting ZIP.
[I] Removing unneeded files.
[I] Adding new files.
[I] Patching XML strings with tags.
[D] Replacing strings in XML data.
[D] Writing to temporary file: D:\XperiaTool.\data\assets\AndroidManifest.xml.t
mp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: D:\XperiaTool.\data\assets\ZPAK\metadata.xml.tmp

[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: D:\XperiaTool.\data\res\values\strings.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: D:\XperiaTool.\data\ZPAK\metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Done with string replacement.
[I] Done.
[I] PSXperiaTool initialized, outputting to: D:\XperiaTool.\output
[I] Starting build.
[I] Checking to make sure all files are there.
[D] Done checking data.
[I] Creating temporary directory.
[D] Created temporary directory at, ..psxperia.678
[I] Copying icon if needed.
[I] Replacing strings.
[D] Replacing strings in XML data.
[D] Writing to temporary file: ..psxperia.678\assets\AndroidManifest.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ..psxperia.678\assets\ZPAK\metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ..psxperia.678\res\values\strings.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ..psxperia.678\ZPAK\metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Done with string replacement.
[D] Done replacing strings.
[I] Generating PSImage.
[I] Beginning PSImage compression.
[I] Generating image TOC.
[D] Done generating PSImage
[I] Generating ZPAK.
[I] Generating zpak file from directory ..psxperia.678\ZPAK with compression =
false
[D] Done with ZPAK creation.
[D] Done generating ZPAK at ..psxperia.678\NCXA00000_1.zpak
[I] Generating default ZPAK.
[I] Generating zpak file from directory ..psxperia.678\assets\ZPAK with compres
sion = true
[D] Done with ZPAK creation.
[D] Done generating default ZPAK at ..psxperia.678\assets\NCXA00000_1.zpak
[I] Done processing, generating output.
[D] Moving files around.
[D] Running command: [.\aapt-windows.exe package -f -F D:\XperiaTool.\output\co
m.sony.playstation.NCXA00000_1.apk.unsigned -S ..psxperia.678\res -M ..psxperi
a.678\assets\AndroidManifest.xml -I .\android-framework.jar ..psxperia.678]
[I] Signing apk D:\XperiaTool.\output\com.sony.playstation.NCXA00000_1.apk.unsi
gned to D:\XperiaTool.\output\com.sony.playstation.NCXA00000_1.apk
[I] Done.
[I] APK file: D:\XperiaTool.\output\com.sony.playstation.NCXA00000_1.apk
[I] Data dir: D:\XperiaTool.\output\data\com.sony.playstation.NCXA00000_1\files
\content
[I] Deleting temporary directory
[I] Done.

**Comment by shotgunfool** [comment]
Still getting cannot build APK error

going to try to build without an image

*Update

No error without an image, copying files to SD card to try and install will post back shortly

*Update

APK installs fine  And RUNS EVEN BETTER :)

**Comment by Levistras** [comment]
Oddly, immediately after install of the APK (after copying over the data folder), the App forced closed.  After the force close I checked the /android/data/ folder on the phone, and the files within /content/ were removed.

Rebooting the phone.. I must have prepared and loaded my copy of Final Fantasy 7 at least 20 times, I think it's getting a little confused.  Started to see some old folder names in /android/data/ that I've created and deleted that are again showing up empty.

After reboot, I sideloaded the APK and the Data folder.

I confirmed (via Astro on handset) that the Data folder is there and the files inside are present.

I installed the APK, immediately after APK install on the phone however, I got a "process com.sony.playstation.NCXA00000_1" has stopped unexpectedly.

I then go back into Astro, and my Data folder is still there, but the ~500mb within is gone, and the "content" folder has been emptied.  The folder structure is there, but the files within are gone.

**Comment by yifanlu** [comment]
That's why I recommend using a unique id. Just copy the data over again and it should work.

**Comment by ChrisKenison** [comment]
I'm getting the same issue shotgunfool was getting.

I didn't get any issues with the Extract or the convert, and when I try to install the apk it works. But, when I try to run the apk, it just closes and goes to the Xperia Play home screen. 

I'm using Windows XP and I used complete default information on the Convert tab. I didn't change anything. Also, this is the only apk I've tried to get working. I don't have any log files. Do you need those if everything on the PSXperia Tool ran smoothly?

Thanks,
Chris

**Comment by Levistras** [comment]
Data copying back over now...  I've yet to be able to get past the app shutting down after the "Updating..." dialog.   Going to try rebuilding, from command prompt, with a serial that isn't "00000" and will repost with logs.

**Comment by yifanlu** [comment]
Can you open a new issue please? I think shotgunfool's issue is already resolved. Also, please post your converter output and android logcat in your issue. (Both of you)
